pkgs/tools/security/fido2luks/default.nix

```
@@ -10,37 +10,28 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "fido2luks";
-  version = "0.2.20";
+  version = "0.2.21";
 
   src = fetchFromGitHub {
     owner = "shimunn";
     repo = pname;
     rev = version;
-    sha256 = "04gl7wn38f42mapmkf026rya668vvhm03yi8iqnz31xgggbr2irm";
+    sha256 = "sha256-bXwaFiRHURvS5KtTqIj+3GlGNbEulDgMDP51ZiO1w9o=";
```

I was somewhat under the impression that there was a preference towards using the `hash` attribute over `sha256`. Why not use it here?
```suggestion
    hash = "sha256-bXwaFiRHURvS5KtTqIj+3GlGNbEulDgMDP51ZiO1w9o=";
```

---

sha256 is still preferred as some programs like `nixpkgs-update` don't know how to handle it.

---

I rescind my suggestion then